Makefile

```diff
@@ -383,6 +403,7 @@ define print-target
     @printf "Executing target: \033[36m$@\033[0m\n"
 endef
 
+release-bundle-GOOS: cmd/konvoy-image-wrapper/image/konvoy-image-builder.tar.gz
```

Comment:
I moved it there because i just wanted to create bundle for only darwin.
However while fixing it, i realized that this target is not used and duplicate of `release-snapshot` target which uses goreleaser to create release bundles. Should we remove this target instead of fixing it and rely on single mechanism that creates release bundles using goreleaser. wdyt?